lib/middleman-cdn/cdns/cloudflare.rb

```diff
@@ -23,6 +23,7 @@ def self.example_configuration_elements
       end
 
       def invalidate(options, files, all: false)
+        options[:invalidate_zone_for_many_files] = true unless options.has_key?(:invalidate_zone_for_many_files)
```

Comment:
Good catch! I actually googled `has_key?` before adding it here but couldn't find any indication that it's deprecated: http://ruby-doc.org/core-2.3.0/Hash.html#method-i-has_key-3F
Still, if Matz says so. :smile: I'll replace it.